tests/foreman/api/test_errata.py

```diff
@@ -35,36 +178,106 @@ def test_positive_install(self):
 
         @Steps:
 
-        1. PUT /katello/api/hosts/bulk/update_content
+        1. PUT /api/v2/hosts/bulk/update_content
 
         @Assert: errata is installed in the host-collection.
 
-        @caseautomation: notautomated
+        @CaseLevel: System
+        """
+        with VirtualMachine(distro='rhel71') as client1, VirtualMachine(
+                distro='rhel71') as client2:
+            clients = [client1, client2]
+            for client in clients:
+                client.install_katello_ca()
+                result = client.register_contenthost(
+                    self.org.label, self.activation_key.name)
+                self.assertEqual(result.return_code, 0)
+                client.enable_repo(REPOS['rhst7']['id'])
+                client.install_katello_agent()
+            host_ids = [
+                entities.Host().search(query={
+                    'search': 'name={0}'.format(client.hostname)})[0].id
+                for client in clients
+                ]
+            self._install_package(clients, host_ids, FAKE_1_CUSTOM_PACKAGE)
+            entities.Host().install_content(data={
+                'organization_id': self.org.id,
+                'included': {'ids': host_ids},
+                'content_type': 'errata',
+                'content': [CUSTOM_REPO_ERRATA_ID],
+            })
+            self._validate_package_installed(clients, FAKE_2_CUSTOM_PACKAGE)
+
+    @tier3
+    def test_positive_install_in_host(self):
+        """Install errata in a host
+
+        @id: 1e6fc159-b0d6-436f-b945-2a5731c46df5
+
+        @Setup: Errata synced on satellite server.
+
+        @Steps:
+
+        1. PUT /api/v2/hosts/:id/errata/apply
+
+        @Assert: errata is installed in the host.
 
         @CaseLevel: System
         """
+        with VirtualMachine(distro='rhel71') as client:
+            client.install_katello_ca()
+            result = client.register_contenthost(
+                self.org.label, self.activation_key.name)
+            self.assertEqual(result.return_code, 0)
+            client.enable_repo(REPOS['rhst7']['id'])
+            client.install_katello_agent()
+            host_id = entities.Host().search(query={
+                    'search': 'name={0}'.format(client.hostname)})[0].id
+            self._install_package([client], [host_id], FAKE_1_CUSTOM_PACKAGE)
+            entities.Host(id=host_id).errata_apply(data={
+                'errata_ids': [CUSTOM_REPO_ERRATA_ID]})
+            self._validate_package_installed([client], FAKE_2_CUSTOM_PACKAGE)
 
-    @stubbed()
     @tier3
     def test_positive_list(self):
-        """View all errata specific to an Org
+        """View all errata specific to repository
```

Comment:
There's no way to filter errata by organization via API. However, it's possible to filter errata by repository. So instead of removing stub changed the purpose of the test and implemented it